### ISSUE TITLE
Add a way to track who has the POE adapter

### ISSUE THREAD

**Issue by Andrew-Dickinson** [WillNilges/meshdb#186]
Add a way to track which member has the PoE adapter so that we can figure out who to call when it goes down unexpectedly

From Lydon [in slack](https://nycmesh.slack.com/archives/C06GW3Q92TY/p1707597786827909):

> Could we include a field for "POE Supply" on each node? This could help us keep track of where power sources are, specifically when devices are offline and we can't just look at the device notes. (Including a "Port" field could do this for omni's as well, but I'm thinking mainly for netpowers etc.). Saves scrolling through chat or guessing who was the first install and should be on port 1. We'd have to keep it up to date, but maybe that's something that could be included in a "post install" form that installers fill out.

**Comment by Andrew-Dickinson** [comment]
Some wrinkles: this could be more than one person in the case of a Netpower install

**Comment by WillNilges** [comment]
This should be covered by the new model through `powered_by_install`.

https://wiki.mesh.nycmesh.net/books/services-software/page/meshdb-schema-design

**Comment by Andrew-Dickinson** [comment]
Perhaps we should also track which port on which device a given install is connected to, I could see an optional pair of fields, one a foreign key to device, the other a string field for the port name / number working well?

**Comment by Andrew-Dickinson** [comment]
https://nycmesh.slack.com/archives/C06GW3Q92TY/p1724821395717539

**Comment by WillNilges** [comment]
https://nycmesh.slack.com/archives/C06GW3Q92TY/p1733245076184009

**Comment by Andrew-Dickinson** [comment]
I could see a celery worker doing that 🤔

**Comment by OlivierMorf** [comment]
Maybe a good idea.
I am afraid installer won't populate on a regular basis that field. Can this been pulled form a snmp query ?
 - Trying to lower the need of manual intervention -
the devices should be labeled properly as well which is not always done.

**Comment by LydonThorpe** [comment]
Yeah unless this can be automated, I think we need to limit the amount of info we track to the bare minimum. Otherwise it will just end up out of date and useless. People will forget to update if things get moved around.

Perhaps the other option is to add a "has POE" check box for install or member objects?

For a single omni install this would mark the 1 member who has the POE injector. For installs with multiple 7rs this would mark all members who have POEs.
